code/datums/brain_damage/split_personality.dm

```diff
@@ -269,6 +269,7 @@
 	poll_time = 10 SECONDS
 	/// Duration of effect, tracked in seconds, not deciseconds. qdels when reaching 0.
 	var/duration_in_seconds = 180
+	var/datum/status_effect/inebriated/inebriation
```

Comment:
You shouldn't need to hold it in a var, it risks hard deletes (you don't do any reference handling) 

You can instead:

```dm
/datum/brain_trauma/severe/split_personality/blackout/on_gain()
	...
	var/datum/status_effect/inebriated/inebriation = owner.has_status_effect(/datum/status_effect/inebriated)
	inebriation?.iron_liver = TRUE

/datum/brain_trauma/severe/split_personality/blackout/on_lose()
	...
	var/datum/status_effect/inebriated/inebriation = owner.has_status_effect(/datum/status_effect/inebriated)
	inebriation?.iron_liver = FALSE
```

And it should give you the same results